validation fails on Solaris

build/i86/doc/dbus-daemon.1.xml:1248: parser error : Double hyphen within comment: <!-- no SELinux or AppArmor support in Solaris
<itemiz
    with <command>systemd --user</command>. This is set up by

--- dbus-1.16.2/doc/dbus-daemon.1.xml.in.orig
+++ dbus-1.16.2/doc/dbus-daemon.1.xml.in
@@ -1261,7 +1261,7 @@
     activation</firstterm>, a mechanism in which the
     <command>dbus-daemon</command> can launch the service on-demand,
     either by running the session service itself or by communicating
-    with <command>systemd --user</command>. This is set up by
+    with <command>systemd user</command>. This is set up by
     creating a <firstterm>service file</firstterm> in the directory
     <filename><replaceable>${datadir}</replaceable>/dbus-1/services</filename>,
     for example:
